packages/block-library/src/navigation-link/edit.js

```diff
@@ -99,15 +100,22 @@ const useIsInvalidLink = ( kind, type, id ) => {
 	const isPostType =
 		kind === 'post-type' || type === 'post' || type === 'page';
 	const hasId = Number.isInteger( id );
+	const blockEditingMode = useBlockEditingMode();
+
 	const postStatus = useSelect(
 		( select ) => {
 			if ( ! isPostType ) {
 				return null;
 			}
+
+			if ( blockEditingMode === 'disabled' ) {
```

Comment:
I would add a comment here as to why we don't want to run the check in the disabled mode 🤔

Reply:
Let's will add that. Just wanted to check if any e2e tests were failing first.